umake/frameworks/ide.py

```diff
@@ -629,7 +629,7 @@ def __init__(self, category, flavour=""):
                          download_page="https://netbeans.org/downloads/zip.html",
                          dir_to_decompress_in_tarball="netbeans*",
                          desktop_filename="netbeans{}.desktop".format(flavour),
-                         packages_requirements=['openjdk-7-jdk', 'jayatana'],
```

Comment:
jayatana is not really required for netbeans, and I'm removing it in line with the other IDEs to avoid possible problems with it (which IIRC is the reason because it's not enabled in IntelliJ and Eclipse)